comps/nginx/nginx.conf.template

```diff
@@ -43,6 +43,10 @@ server {
         proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_connect_timeout 6000;
+        proxy_send_timeout 6000;
+        proxy_read_timeout 6000;
+        send_timeout 6000;
```

Comment:
Are these seconds? `6000` seconds is like `100` minutes.
Defaults are usually `60` seconds. Also don't you need to have `s` after the number like `6000s`?

Reply:
validated by @ichbinblau. Some timeouts were tuned based on large scale testing